director/static/style.css

```diff
@@ -101,4 +101,8 @@
   box-shadow: none;
   font-weight: normal;
   font-size: 90%;
-}
\ No newline at end of file
+}
+
+.app-logo {
+  filter: drop-shadow(0 0 0.5rem rgb(255, 255, 255));
+}
```

Comment:
I'm sorry but I'm really not fan of this new style... Can you try to improve it? For instance maybe changing the app-logo class filter.

@anthonyolea what's your opinion on that? Is-it ok for you? (maybe I'm not the best to validate or not UI styles ^^)

![image](https://user-images.githubusercontent.com/1552526/163228149-43382f97-078b-4f14-9562-489fe27c5039.png)

Reply:
Also there is a dark mode, I do not know if you tried with this one.
But, anyway, I fear this kind of shadow is a little bit odd with the global look'n'feel.

IMO, I can fairly understand the need to customize the UI. However I am not really enthusiast to have a single key in the configuration matching a specific UI element to customize. It feels very limited and harder to extend.
I guess one proper way to achieve this could be to offer extend or override the default templates and/or stylesheet? Then you may have to set a flag in the global configuration to enable this.

Reply:
Yes, it seems to be a better approach to override template. Let me open a new merge request for this feature

Reply:
@PaulinCharliquart In this case can you please remove from this current PR the code related to the customized logo please? We'll be able to review & merge it :)